### DISCUSSION TITLE
Am I missing something?

### DISCUSSION THREAD

**Discussion by Meraroo** [ANLAB-KAIST/KENSv3#29] [Project Q&A 1. Application Layer (Socket)] [answered]
I just write simple code, but the code is blocked..
![image](https://user-images.githubusercontent.com/81629557/159017429-408a1798-4842-4216-ac03-64fcb1a22a90.png)
![image](https://user-images.githubusercontent.com/81629557/159017461-f3d1da76-5ca1-4ca6-afc6-b02a83b04626.png)
The code is stopped and, not working.. When I use socket() function, the same thing always happens..

**Comment by Pusnow** [comment, marked as answer]
The last parameter of the `socket()` call should be `IPPROTO_TCP`.

I'm not sure why the socket call is blocking. It might be a KENS's bug. Sorry for the inconvenience.

**Reply by Meraroo** [reply]
The problem was the third parameter! Thanks for your advice
